apps/web-client/src/routes/orders/suppliers/reconcile/[id]/+page.svelte

```diff
@@ -167,10 +163,16 @@
 				</nav>
 
 				{#if currentStep === 1}
-					<form class="flex w-full gap-2" use:enhance method="POST">
+					<form
+						class="flex w-full gap-2"
+						on:submit|preventDefault={() => {
```

Comment:
This `on:submit` event handler is exposed by svelte, it is not connected to superforms. If we are going to use superforms, we should use it as I have set it up (diff on line 48 -> 57) otherwise we aren't gettting any benefits of the validation that occurs in their [event chain](https://superforms.rocks/concepts/events). The input should already be cleared on every successful submission - if its not behaving, there are other ways to fix it

Reply:
?

Reply:
If you want to keep the input focussed after submit, see what I have done in `apps/web-client/src/routes/orders/customers/[id]/+page.svelte` of my `fix/customer-order-line` PR